containers/podman-desktop
# feat: update forms look & feel

### What does this PR do?

Updates the palette + minor layout details across the following forms to be more consistent with the rest of the UI in accordance with the updated palette from https://github.com/containers/podman-desktop/pull/2223 :

- deploy to k8s
- play kube
- pull image
- build image

### Screenshot/screencast of this PR

#### deploy to k8s
<table>
<tr>
<th>before</th>
<th>after</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-01 16-33-20](https://user-images.githubusercontent.com/799683/235526239-ccbfc96f-0046-4fb9-93d0-ac6c3776bb95.png)

</td><td>

![Screenshot from 2023-05-01 16-02-58](https://user-images.githubusercontent.com/799683/235522388-7264df24-777f-416b-8950-3f126850a445.png)

</td>
</tr>
</table>

#### play kube

<table>
<tr>
<th>before</th>
<th>after</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-01 16-32-34](https://user-images.githubusercontent.com/799683/235527197-95c93302-01bc-4b6e-8e49-0f6685190ee4.png)

</td><td>

![Screenshot from 2023-05-01 15-56-22](https://user-images.githubusercontent.com/799683/235522307-9f17a3d9-54db-42cc-b59e-8b2c4b17f88a.png)

</td>
</tr>
</table>


#### pull image

<table>
<tr>
<th>before</th>
<th>after</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-01 16-32-42](https://user-images.githubusercontent.com/799683/235527122-89ab8737-ae85-4092-bb42-0406a65e5f58.png)

</td><td>

![Screenshot from 2023-05-01 15-48-44](https://user-images.githubusercontent.com/799683/235522326-4a5f05e6-797e-48eb-a3d3-7dea196e2762.png)

</td>
</tr>
</table>

#### build image

<table>
<tr>
<th>before</th>
<th>after</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-01 16-32-47](https://user-images.githubusercontent.com/799683/235527271-9bb37e9f-c814-4844-85b7-a352ef95ff49.png)

</td><td>

![Screenshot from 2023-05-01 15-44-50](https://user-images.githubusercontent.com/799683/235522350-c5ccdf10-f790-4b13-9acd-34dd41bcb2dd.png)

</td>
</tr>
</table>

### What issues does this PR fix or reference?

https://github.com/containers/podman-desktop/issues/2156

### How to test this PR?

Run through each action in the UI (deploy to k8s, play kube, pull image, build image) and make sure the forms look ok. No global stylesheet changes / overrides were made so it shouldn't affect anything outside of the named forms.
